### PR TITLE
nhc98: fixing build on >10.6

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

An initial attempt to fix build on later systems.
Port remains restricted to 32-bit OSs, since that is exactly what its configure script has.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6 PPC
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by barracuda156** [comment]
@mascguy With this I also need your help to test it on build bots: we need 32-bit OS, so `${os.major} > 10 && ${os.major} < 19`.

**Comment by mascguy** [comment]
> @mascguy With this I also need your help to test it on build bots: we need 32-bit OS, so `${os.major} > 10 && ${os.major} < 19`.

I only have 32-bit installations, on two old laptops (one PPC 10.4/10.5, the other Intel 10.5/10.6).

So I may not be able to get to this until later today, or sometime tomorrow.

File: lang/nhc98/Portfile [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-name        nhc98
-version     1.22
-revision    0
-categories  lang
-maintainers nomaintainer
+name            nhc98
+version         1.22
+revision        1
+categories      lang
+maintainers     Sergey Fedorov @barracuda156
```
Comment (reneeotten) [inline lang/nhc98/Portfile:9]: this is an incorrect `maintainers` line; it should provide an (obfuscated) e-mail address and a GitHub handle, and ideally also `openmaintainer`

File: lang/nhc98/Portfile [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
+PortSystem      1.0
 
-name        nhc98
-version     1.22
-revision    0
-categories  lang
-maintainers nomaintainer
+name            nhc98
+version         1.22
+revision        1
```
Comment (reneeotten) [inline lang/nhc98/Portfile:7]: a `revision` increase is unlikely to be required when you're trying to fix build issues as the PR description mentions

File: lang/nhc98/Portfile (hunk 1 above)
Comment (barracuda156) [reply lang/nhc98/Portfile:9]: @reneeotten Thank you! How to write e-mail (gmail) and Github handle correctly?
And you are right, `openmaintainer` should be added, I just forgot about it.

File: lang/nhc98/Portfile (hunk 2 above)
Comment (barracuda156) [reply lang/nhc98/Portfile:7]: Fair enough.

**Comment by barracuda156** [comment]
@reneeotten Both suggested changed now done.

**Comment by barracuda156** [comment]
@mascguy This can be merged, perhaps. Either we get some additional systems fixed (hopefully) or not, but it cannot break anything.

**Review by mascguy — APPROVED** [review]

**Pull request merged**